### PR TITLE
Missing publications as

### PR DESCRIPTION
**Pull request by AlexanderSchultheiss**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BRSK:JSS24,
+  author       = {Alexander Boll and
+                  Pooja Rani and
+                  Alexander Schulthei{\ss} and
+                  Timo Kehrer},
+  title        = {{Beyond code: Is there a difference between comments in visual and
+                  textual languages?}},
```
Comment (pmbittner) [inline literature.bib:346]: capitalization wrong: https://capitalizemytitle.com/

File: literature.bib [hunk 2]
```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BRSK:JSS24,
+  author       = {Alexander Boll and
```
Comment (pmbittner) [inline literature.bib:341]: please but authors into a single line for consistency

File: literature.bib [hunk 3]
```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
...
+                  Pooja Rani and
+                  Alexander Schulthei{\ss} and
+                  Timo Kehrer},
+  title        = {{Beyond code: Is there a difference between comments in visual and
+                  textual languages?}},
+  journal      = JSS,
+  volume       = 215,
+  pages        = {112087},
```
Comment (pmbittner) [inline literature.bib:349]: Does your paper really have that many pages? :sweat_smile:  Is there a `-` missing somewhere?

File: literature.bib (hunk 2 above)
Comment (pmbittner) [reply literature.bib:341]: also for titles, and also for the other new entry

**Review by pmbittner — CHANGES_REQUESTED** [review]

File: literature.bib (hunk 1 above)
Comment (AlexanderSchultheiss) [reply literature.bib]: done

File: literature.bib (hunk 2 above)
Comment (AlexanderSchultheiss) [reply literature.bib]: done

File: literature.bib (hunk 3 above)
Comment (AlexanderSchultheiss) [reply literature.bib:345]: This number also struck me as crazy, but upon checking it I found the same number on Elsevier's website.

File: literature.bib (hunk 1 above)
Comment (AlexanderSchultheiss) [reply literature.bib]: Previously, I used the official capitalization of the Journal. I am aware of the website that you linked.

Comment (pmbittner) [reply literature.bib]: Ok, we never use the official capitalization. ACM also uses lower case titles.

File: literature.bib (hunk 3 above)
Comment (pmbittner) [reply literature.bib:345]: Elsevier, a pain in the a** as always.

**Review by pmbittner — APPROVED** [review]

File: literature.bib (hunk 3 above)
Comment (AlexanderSchultheiss) [reply literature.bib:345]: indeed

**Pull request merged**